# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/libcubeb/src/android/cubeb-output-latency.h
# Commit: 9794b0a3bffc
# Full Hash: 9794b0a3bffc474951b33527b738488717caffbf
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-12-12 21:50:40
# Regressor Bug: 1603384
# File Overlap Count: 1
# Description:
#   Bug 1603384 - Update cubeb from upstream to aa63601. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56872
# ==============================================================================

diff -r 7d77d727cd1e -r 9794b0a3bffc media/libcubeb/src/android/cubeb-output-latency.h
--- a/media/libcubeb/src/android/cubeb-output-latency.h	Thu Dec 12 17:46:46 2019 +0200
+++ b/media/libcubeb/src/android/cubeb-output-latency.h	Thu Dec 12 15:24:23 2019 +0000
@@ -35,7 +35,7 @@
 bool
 cubeb_output_latency_method_is_loaded(output_latency_function * ol)
 {
-  assert(ol && (ol->from_jni || ol->from_lib));
+  assert(ol);
   if (ol->version > ANDROID_JELLY_BEAN_MR1_4_2){
     return !!ol->from_jni;
   }